lib/api_repo.js

```diff
@@ -434,7 +440,7 @@ function collectProtoDeps(includePath, protoFile, done) {
   args.push(protoFile);
   // Invokes protoc with --dependency_out to generate the dependency
   // proto files.
-  execFile('protoc', args, {}, function(err) {
+  execFile(this.protoCompiler, args, {}, function(err) {
```

Comment:
Unfortunately `collectProtoDeps` is not a method, `this.` wouldn't work well. Pass it as an argument, or modify the function itself ot a method.